### DISCUSSION TITLE
Best Arrangement

### DISCUSSION THREAD

**Discussion by th3ba1r4m** [Security-Onion-Solutions/securityonion#4768] [Unsupported Versions] [answered]
First of all I want to say a lot thanks for prepared so good product like that.

I have some questions relating with improving my experience SO  product and selecting good structure type to my big large enterprise.

1) I have a thought : disable SO default alerts then only playbook logs activate and  I dont know what is good or bad sides doing it. Or using both of them together how much resource(system requirements)  I have need for not slowly working  ? 

2) I want to  establish my system as seperated Manager,Search and one Forward node.But again I don't know  what will best system requirement choosement for each of them .And  one forward node will enough ? 

3) Last questions is how I can send sysmon logs to SO ? I cant see them on  alerts or hunting interface. What is your advice for sending sysmon logs ? 

Thanks on advance

Waiting impatiently to Securty Onion conference.

**Comment by weslambert** [comment, marked as answer]
To address your questions:

(1) I would leave the alerts as-is, then tune them out and add  plays as you see fit.
(2) The number of forward nodes depends on what segments you intend to monitor, as well as how much traffic you intend to mirror to Security Onion.
(3) You can pick up Sysmon logs using Osquery, Winlogbeat, or Wazuh.  We support all of these methods.
